### PR TITLE
refine example for SQ and WOQ

### PR DESCRIPTION
**Pull request by xin3he**
## Type of Change

example refine

## Description

add test for gptq and teq, refine workflow for ease-of-use

### CODE REVIEW

File: examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py [hunk 1]
```diff
@@ -58,11 +45,28 @@
...
+# ============SmoothQuant configs==============
+parser.add_argument("--sq", action="store_true")
+parser.add_argument("--alpha", default="auto", help="Smooth quant parameter.")
+# ============WeightOnly configs===============
+parser.add_argument("--weight_only_algo", default="RTN", choices=['RTN', 'AWQ', 'TEQ', 'GPTQ'], 
+                    help="Weight-only parameter.")
 parser.add_argument("--weight_only_bits", type=int, default=8)
 parser.add_argument("--weight_only_group", type=int, default=-1)
 parser.add_argument("--weight_only_scheme", default="sym")
+parser.add_argument("--weight_only_mse_range", action="store_true")
```
Comment (mengniwang95) [inline examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py:58]: weight_only_mse_range only takes effect for RTN according to below code,  but its name seems to say it takes effect for the whole weight_only

**Review by YIYANGCAI — APPROVED** [review]

File: examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py (hunk 1 above)
Comment (xin3he) [reply examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py:58]: good point

Comment (hshen14) [reply examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py:58]: Is it better to rename "weight_only_bits" => woq_bits?

Similarly, woq_group_size, woq_scheme, woq_rtn_mse?

What's the meaning of mse_range, mse driven RTN?

Comment (xin3he) [reply examples/huggingface/pytorch/language-modeling/quantization/run_clm_no_trainer.py:58]: okay, thank you.

**Comment by xin3he** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1289/

**Review by hshen14 — APPROVED** [review]

**Comment by xin3he** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1299/

**Comment by VincyZhang** [comment]
seems extension test passed, please check result: https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1304/artifact/report.html

**Pull request merged**